src/components/Projects/ProjectTabs.js

```diff
@@ -29,47 +29,45 @@ const ProjectTabs = ( ): Node => {
     optsWithAuth => searchProjects( apiParams, optsWithAuth )
   );
 
-  const fetchProjectsByLatLng = ( ) => {
-    setApiParams( {
-      lat: latLng.latitude,
-      lng: latLng.longitude
-    } );
-  };
-
-  const fetchFeaturedProjects = ( ) => setApiParams( { features: true } );
-  const fetchUserJoinedProjects = useCallback( ( ) => setApiParams(
-    { member_id: memberId }
-  ), [memberId] );
-
   useEffect( ( ) => {
-    if ( memberId ) {
-      fetchUserJoinedProjects( );
+    if ( memberId && currentTabId === JOINED_TAB_ID ) {
+      setApiParams( { member_id: memberId } );
+    } else if ( currentTabId === FEATURED_TAB_ID ) {
+      setApiParams( { features: true } );
+    } else if ( currentTabId === NEARBY_TAB_ID ) {
+      setApiParams( {
+        lat: latLng.latitude,
+        lng: latLng.longitude
+      } );
```

Comment:
Moved these to the `useEffect` because I found in some instances that you can be on one tab, but different data shows. Usually happened when navigating away from the projects page and back.

Now the query is a reflection of the current tab.